packages/core/strapi/lib/commands/admin-create.js

```diff
@@ -95,14 +94,17 @@ async function createAdmin({ email, password, firstname, lastname }) {
   const appDir = process.cwd();
 
   const isTSProject = await tsUtils.isUsingTypeScript(appDir);
+  const compiledDirectoryPath = isTSProject
```

Comment:
```suggestion
  const outDirPath = isTSProject
```
Maybe consider rewording this variable so that it's clear we're talking about the out directory here? WDYT?